### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by Dilden**
This is an initial attempt at adding Docker support to the project to make standing up a development environment even faster. Addresses issue #3

### CODE REVIEW

**Comment by Dilden** [comment]
This should also include a node container at some point. For now, I've followed [this blog post](https://faun.pub/managing-mongodb-on-docker-with-docker-compose-26bf8a0bbae3). [Here is the official documentation](https://hub.docker.com/_/mongo/) for the container images used.

**Comment by Dilden** [comment]
This PR addresses #3

**Comment by Dilden** [comment]
This request now includes everything needed to get the whole environment up and running within docker. No need to install MongoDB or even Node on your own machine.

File: README.md [hunk 1]
```diff
@@ -1,54 +1,65 @@
...
+To get your Perspective API token, follow the [getting started guide](https://developers.perspectiveapi.com/s/docs-get-started), and then follow the "Enable the API" section.
 
-### Create a filed named `.env` in the root directory
+## Start w/Docker
+To simplify getting started with this bot, a `docker-compose.yml` file has been included. Ensure you have installed [docker/docker-compose](https://docs.docker.com/get-docker/). Once installed, run `docker-compose up -d` to start the environment. This will do a few things:
+1. Download and install images for Node 16, MongoDB, and Mongo Express (web interface for managing MongoDB)
+2. Install required JavaScript dependencies
+3. Setup [ngrok](https://ngrok.com/) tunnel
+4. Set the webhook of your bot to the ngrok tunnel.
+5. Start [nodemon](https://nodemon.io/) to watch TypeScript files, buil on save, and restart the application.
```
Comment (jimmyhogoboom) [inline README.md:40]: "build on save"

**Review by jimmyhogoboom — CHANGES_REQUESTED** [review]
This is awesome! Once I managed to get `docker-compose` working, getting it running is EZ. Just found a couple little typos, But then this is good to go!

**Review by jimmyhogoboom — APPROVED** [review]

**Pull request merged**